### PR TITLE
Create blog page

### PR DESCRIPTION
**Pull request by NARCISA-LUCIA**

### CODE REVIEW

File: src/app/blog/blog.component.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+mat-divider {
+  background-color: #526afe;
+  width: 35px;
+  border-width: 3px;
+  margin-bottom: 50px;
+}
+
+.container-image-blog-shadow>:first-child{
+  /* border:1px solid red;*/
```
Comment (sabaudragos) [inline src/app/blog/blog.component.css]: Could you pls remove the comment and the extra line?

File: src/app/blog/blog-card/blog-card.component.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
+
```
Comment (sabaudragos) [inline src/app/blog/blog-card/blog-card.component.css:4]: Could you check the styling of this file? There a space before each line of code.

File: src/app/blog/blog-card/blog-card.component.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  .title-blog {
+    color: #6c6c6c;
+
```
Comment (sabaudragos) [inline src/app/blog/blog-card/blog-card.component.css]: Could you please remove this empty line?

File: src/app/blog/blog-card/blog-card.component.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  .title-blog {
+    color: #6c6c6c;
+   
+  }
+  
+  .text-blog {
+    padding: 10px;
+
```
Comment (sabaudragos) [inline src/app/blog/blog-card/blog-card.component.css]: This one here too. Empty line.

File: src/app/blog/blog-card/blog-card.component.css [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  h5 {
+    color: #565656;
+    font-size: 22px;
+  }
+  
+  .paragraph-long {
+    color: #acacac;
+    letter-spacing: 0.1px;
+    line-height: 1.6;
+
```
Comment (sabaudragos) [inline src/app/blog/blog-card/blog-card.component.css]: And this one

**Review by sabaudragos — CHANGES_REQUESTED** [review]

**Comment by sabaudragos** [comment]
I've left some minor comments. The PR as a whole looks pretty good.

File: src/app/blog/blog.component.css (hunk 1 above)
Comment (NARCISA-LUCIA) [reply src/app/blog/blog.component.css]: Done

File: src/app/blog/blog-card/blog-card.component.css (hunk 2 above)
Comment (NARCISA-LUCIA) [reply src/app/blog/blog-card/blog-card.component.css:4]: Done

File: src/app/blog/blog-card/blog-card.component.css (hunk 4 above)
Comment (NARCISA-LUCIA) [reply src/app/blog/blog-card/blog-card.component.css]: Done

File: src/app/blog/blog-card/blog-card.component.css (hunk 5 above)
Comment (NARCISA-LUCIA) [reply src/app/blog/blog-card/blog-card.component.css]: Done

**Review by sabaudragos — APPROVED** [review]

**Pull request merged**